WordPress/Classes/ViewRelated/System/Sidebar/SidebarViewModel.swift

```diff
@@ -50,6 +50,15 @@ final class SidebarViewModel: ObservableObject {
             .publisher(for: .init(rawValue: WordPressAuthenticator.WPSigninDidFinishNotification))
             .sink { [weak self] _ in self?.resetSelection() }
             .store(in: &cancellables)
+
+        NotificationCenter.default
+            .publisher(for: .WPAccountDefaultWordPressComAccountChanged)
+            .receive(on: DispatchQueue.main)
```

Comment:
(nit) `receive(on` adds a thread hop. I'd remove it unless it's necessary.

Reply:
My thinking is that the notification handling code does not need to execute in the same runloop as the the notification firing code, but does need to execute on the main thread.